### PR TITLE
{vis}[GCCcore/8.2.0] gsettings-desktop-schemas v3.34.0

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'MesonNinja'
+
+name = 'GSettings-Desktop-Schemas'
+version = '3.34.0'
+
+homepage = 'https://gitlab.gnome.org/GNOME'
```
Comment (smoors) [inline easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: @Flamefire this page is probably more appropriate?
https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
+easyblock = 'MesonNinja'
+
+name = 'GSettings-Desktop-Schemas'
```
Comment (smoors) [inline easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: @Flamefire it looks like the lowercase name `gsettings-desktop-schemas` is used officially, so let's use that?

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: Oh you are right, just copied half of it :/

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb (hunk 2 above)
Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: I was under the impression that recent ECs tend to use Capitalized names and wanted to be consistent with that. I don't mind changing this if I'm mistaken.

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb (hunk 1 above)
Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: Ah, I got the URL from https://github.com/GNOME/gsettings-desktop-schemas

File: easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb (hunk 2 above)
Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: @boegel What do you say?

Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: Comparison example:
- https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/blob/master/gsettings-desktop-schemas.pc.in "Name: gsettings-desktop-schemas"
- https://gitlab.gnome.org/GNOME/gobject-introspection/blob/master/gobject-introspection-no-export-1.0.pc.in "Name: gobject-introspection"

The latter is capital cased in EB

Comment (boegel) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: We try to stick to the "official" software name.

In this case, compared https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas/blob/master/README with https://gitlab.gnome.org/GNOME/gobject-introspection/blob/master/README.rst...

Comment (Flamefire) [reply easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb]: Done

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/a57df7ad2d8cd130bdfb3c7ff9395050 for a full test report.

**Comment by boegel** [comment]
@Flamefire Doesn't work for me due to:

```
meson: error: unrecognized arguments: --disable-silent-rules
```

**Comment by Flamefire** [comment]
Strange... must be crept in when rebasing or so. Removed :)

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node374.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/bff935b3caaef65e4915f61c0a25c22b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/161a470d3ad65576fdc9382777c43ae9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3144.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/bf9adf14b19c56410759dc3df6d6521c for a full test report.

**Comment by boegel** [comment]
@smoors Good to go (you have a request for changes still in place)?

**Comment by smoors** [comment]
Going in, thanks @Flamefire!

**Pull request merged**